README.md

```diff
@@ -32,8 +32,40 @@ let tcxUrl = URL(fileURLWithPath: "TestFile" + ".tcx")
 let tcxData = try? Data(contentsOf: tcxUrl)
 
 if let tcxData = tcxData {
-    let tcxFile = try? TcxFile.decode(from: tcxData)
-}
+
+do {
+  let tcxFile = try TcxFile.decode(from: tcxData)
+  guard let numActivities = tcxFile.database.activities?.activities?.count else { return  }
+  
+  for activity in 0..<numActivities {    
+    guard let numLaps = tcxFile.database.activities?.activities?[activity].lap.count else {  }
+    for laps in 0..<numLaps {
```

Comment:
vert space between the guard and the loop..

also I would switch to just doing
```
-- If you really want to check Empty but that is not needed if you change to the for in loop
guard let activities = tcxFile.database.activities?.activities, !activities.isEmpty else { return }

for activity in activities {

}
```